composer.json

```diff
@@ -16,6 +16,10 @@
             "name": "Thomas Citharel",
             "homepage": "http://tcit.fr",
             "role": "Developer"
+        },
+        {
+            "name": "Jérémy Besnoit",
```

Comment:
And stop calling me _Besnoit_ when it's *Benoist* :)

_Yes you can also update it in your email client_ :stuck_out_tongue: